### PR TITLE
Fix LatLong.transform on empty dask data

### PR DESCRIPTION
**Pull request by rwedge**
The `LatLong.transform` method had an issue when trying to transform an empty dask series.

The `meta` parameter for the dask `Series.apply` can be a tuple but if so it should be `(name, dtype)`.  The code was using `(series, dtype)`.

I believe this also removes the need to set `series.name` after the apply

### CODE REVIEW

File: woodwork/tests/logical_types/test_logical_types.py [hunk 1]
```diff
@@ -155,6 +155,25 @@ def test_latlong_transform(latlong_df):
         pd.testing.assert_series_equal(actual, expected)
 
 
+def test_latlong_transform_empty_series(empty_latlong_df):
+    df_type = str(type(empty_latlong_df))
+    dask = "dask" in df_type
+    spark = "spark" in df_type
```
Comment (gsheni) [inline woodwork/tests/logical_types/test_logical_types.py:161]:
```suggestion
    is_dask = "dask" in df_type
    is_spark = "spark" in df_type
```

File: woodwork/tests/logical_types/test_logical_types.py [hunk 2]
```diff
@@ -155,6 +155,25 @@ def test_latlong_transform(latlong_df):
...
+
+    latlong = LatLong()
+    series = empty_latlong_df['latlong']
+    actual = latlong.transform(series)
+
+    if dask:
+        actual = actual.compute()
+    elif spark:
```
Comment (gsheni) [inline woodwork/tests/logical_types/test_logical_types.py:169]:
```suggestion
    if is_dask:
        actual = actual.compute()
    elif is_spark:
```

**Review by gsheni — CHANGES_REQUESTED** [review]
Overall, looks good

File: woodwork/tests/logical_types/test_logical_types.py [hunk 3]
```diff
@@ -155,6 +155,25 @@ def test_latlong_transform(latlong_df):
         pd.testing.assert_series_equal(actual, expected)
 
 
+def test_latlong_transform_empty_series(empty_latlong_df):
+    df_type = str(type(empty_latlong_df))
+    is_dask = "dask" in df_type
+    is_spark = "spark" in df_type
```
Comment (jeff-hernandez) [inline woodwork/tests/logical_types/test_logical_types.py:161]: Could we use `_is_dask_dataframe` and `_is_spark_dataframe`?

Comment (rwedge) [reply woodwork/tests/logical_types/test_logical_types.py:161]: what if I change the `if / elif` to `if "dask" in df_type / elif "spark" in df_type`? and remove these variables?

Comment (jeff-hernandez) [reply woodwork/tests/logical_types/test_logical_types.py:161]: Sure, that works too, but are there any drawbacks for using `if _is_dask_dataframe(empty_latlong_df) ... elif _is_spark_dataframe(empty_latlong_df)`?

Comment (rwedge) [reply woodwork/tests/logical_types/test_logical_types.py:161]: it would be fewer lines, but `_is_x_dataframe` will probably be the clearest in meaning

**Review by jeff-hernandez — APPROVED** [review]
LGTM

**Review by gsheni — APPROVED** [review]

**Pull request merged**